scripts/ingests/ingest_bard14_spectra.py

```diff
@@ -0,0 +1,122 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+from astropy.table import Table
+from astropy.io import ascii
+import astropy.units as u
+from astropy.coordinates import Angle
+   
+
+SAVE_DB = False  # True: save the data files(json) in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+#bard14 url : https://docs.google.com/spreadsheets/d/11o5NRGA7jSbHKaTNK7SJnu_DTECjsyZ6rY3rcznYsJk/edit#gid=0
+
+SHEET_ID = '11o5NRGA7jSbHKaTNK7SJnu_DTECjsyZ6rY3rcznYsJk'
+SHEET_NAME = 'bard14'
+full = 'all'
+
+url = f'https://docs.google.com/spreadsheets/d/{SHEET_ID}/gviz/tq?tqx=out:csv&sheet={full}'
+
+bard14_table = ascii.read(
+    url,
+    format="csv",
+    data_start=1,
+    header_start=0,
+    guess=False,
+    fast_reader=False, 
+    delimiter=",",
+)
+
+#print result table
+print(bard14_table.info)
+
+
+#Loop through data and update spectra
+#def updating_spectra(db):
+#for row in bard14_table[0:19]:  
+
+        # Print spectra information
+ #       print("Spectra Information:")
+        
+#        for col_name in row.colnames:
+ #           print(f"{col_name}: {row[col_name]}")
+
+#Call spectra function
+#updating_spectra(db)
+
+#Ingest spectra loop 2
+            
+#def update_all_spectra(db):
+    #updating_spectra(db)
+#    with db.engine.begin() as conn:
+#        for entry in update_all_spectra:
+#            source_value = entry['Source']
+#            spectrum_value = entry['Spectrum']
+#            original_spectrum_value = entry['Original Spectrum']
+          
+#function to update all spectra in the database
+def update_all_spectra(db):
+    with db.engine.begin() as conn:
+        for index, row in bard14_table.iterrows():
+            source_value = row['Source']
+            spectrum_value = row['Spectrum']
+            original_spectrum_value = row['Original Spectrum']          
+
+
+#update data in loop
+        conn.execute(db.Spectra_table.update()\
+                .where(db.Spectra_table.c.source == source_value)\
+                .values(spectrum = spectrum_value))
+        
+        conn.execute(db.Spectra_table.update()\
+                .where(db.Spectra_table.spectrum == spectrum_value)\
+                .values(spectrum = spectrum_value))
+        
+        conn.execute(db.Spectra_table.update()\
+                .where(db.Spectra_table.c.original_spectrum == source_value)\
+                .values(original_spectrum = original_spectrum_value))
```

Comment:
This is not quite right. It's trying to find spectra files that are set to the source name. 
```suggestion
        conn.execute(db.Spectra_table.update()\
                .where(db.Spectra.c.source == source_value)\
                .values(original_spectrum = original_spectrum_value))
```